modin/engines/ray/utils.py

```diff
@@ -148,8 +119,6 @@ def initialize_ray(
                 address=redis_address or "auto",
                 include_dashboard=False,
                 ignore_reinit_error=True,
-                _redis_password=redis_password,
```

Comment:
wait, how do we pass redis password for Ray if we're connecting to an already existing cluster?..

Reply:
It should be added back